extra_tests/snippets/stdlib_ssl_short_recv.py

```
@@ -0,0 +1,84 @@
+import os
+import random
```

Flake8 (7.3.0)

[error] 'random' imported but unused

(F401)